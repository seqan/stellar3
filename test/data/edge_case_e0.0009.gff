1	Stellar	eps-matches	544	1693	99.913	+	.	1;seq2Range=1807,2956;eValue=0;cigar=1150M;mutations=865A
