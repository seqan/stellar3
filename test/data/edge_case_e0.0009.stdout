I/O options:
  database file   : ref.fasta
  query file      : query_e0.0009.fasta
  alphabet        : dna5
  output file     : edge_case_e0.0009.gff
  output format   : gff

User specified parameters:
  minimal match length             : 1000
  maximal error rate (epsilon)     : 0.0009 = (9/10000)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500

Calculated parameters:
  k-mer length : 32
  s^min        : 556
  threshold    : 969
  distance cut : 1032
  delta        : 16
  overlap      : 1

Loaded 1 query sequence.
Loaded 1 database sequence.

All matches resulting from your search have an E-value of: 
        0 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  1
    # SWIFT hits      : 1
    Longest hit       : 1150
    Avg hit length    : 1150
  1, complement

# Eps-matches     : 1
Longest eps-match : 1150
Avg match length  : 1150
